placing_distance: 0.5